tests/fixtures/issues/issue-900.php

```diff
@@ -9221,166 +9221,7 @@
-            'isSyndicationReader' => false,
-            'Crawler' => false,
-            'isFake' => false,
-            'isAnonymized' => false,
-            'isModified' => false,
-            'CssVersion' => '3',
-            'AolVersion' => '0',
-            'Device_Name' => 'Windows Desktop',
-            'Device_Maker' => 'unknown',
-            'Device_Type' => 'Desktop',
-            'Device_Pointing_Method' => 'mouse',
-            'Device_Code_Name' => 'Windows Desktop',
-            'Device_Brand_Name' => 'unknown',
-            'RenderingEngine_Name' => 'WebKit',
-            'RenderingEngine_Version' => 'unknown',
-            'RenderingEngine_Maker' => 'Apple Inc',
-        ],
-        'lite' => false,
-        'standard' => true,
-    ],
-    'issue-900-GC' => [
-        'ua' => 'Mozilla/5.0 (Macintosh; U; Intel Mac OS X 10_6_5; en-US) AppleWebKit/534.16 (KHTML, like Gecko) Chrome/10.0.648.204',
-        'properties' => [
-            'Comment' => 'Chrome 10.0',
-            'Browser' => 'Chrome',
-            'Browser_Type' => 'Browser',
-            'Browser_Bits' => '32',
-            'Browser_Maker' => 'Google Inc',
-            'Browser_Modus' => 'unknown',
-            'Version' => '10.0',
-            'MajorVer' => '10',
-            'MinorVer' => '0',
-            'Platform' => 'MacOSX',
-            'Platform_Version' => '10.6',
-            'Platform_Description' => 'Mac OS X',
-            'Platform_Bits' => '32',
-            'Platform_Maker' => 'Apple Inc',
-            'Alpha' => false,
-            'Beta' => false,
-            'Win16' => false,
-            'Win32' => false,
-            'Win64' => false,
-            'Frames' => true,
-            'IFrames' => true,
-            'Tables' => true,
-            'Cookies' => true,
-            'BackgroundSounds' => false,
-            'JavaScript' => true,
-            'VBScript' => false,
-            'JavaApplets' => false,
-            'ActiveXControls' => false,
-            'isMobileDevice' => false,
-            'isTablet' => false,
-            'isSyndicationReader' => false,
-            'Crawler' => false,
-            'isFake' => false,
-            'isAnonymized' => false,
-            'isModified' => false,
-            'CssVersion' => '3',
-            'AolVersion' => '0',
-            'Device_Name' => 'Macintosh',
-            'Device_Maker' => 'Apple Inc',
-            'Device_Type' => 'Desktop',
-            'Device_Pointing_Method' => 'mouse',
-            'Device_Code_Name' => 'Macintosh',
-            'Device_Brand_Name' => 'Apple',
-            'RenderingEngine_Name' => 'WebKit',
-            'RenderingEngine_Version' => 'unknown',
-            'RenderingEngine_Maker' => 'Apple Inc',
-        ],
-        'lite' => false,
-        'standard' => true,
-    ],
-    'issue-900-GD' => [
-        'ua' => 'Mozilla/5.0 (Windows; U; Windows NT 6.1; ru-RU; AppleWebKit/534.16; KHTML; like Gecko; Chrome/10.0.648.11;Safari/534.16)',
```

Comment:
Oddly formatted in general (no closing parentheses after the platform, semi-colon after the chrome version).